Notebooks/DC2 Reference Catalog Reader.ipynb

```diff
@@ -103,21 +102,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [conda env:3.6-anaconda-4.4]",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "conda-env-3.6-anaconda-4.4-py"
+   "name": "python2"
```

Comment:
Why was this run under Python 2?

This is probably pretty minor, but since we're only guaranteeing that things will work under Python 3, I would suggest that this Notebook be updated to reflect running with a Python 3 kernel.